WooCommerce/Classes/Authentication/Store Creation/StoreCreationCoordinator.swift

```diff
@@ -137,12 +136,47 @@ private extension StoreCreationCoordinator {
             self.navigationController.dismiss(animated: true)
         }
     }
+
+    func showDiscardChangesAlert() {
+        let alert = UIAlertController(title: Localization.DiscardChangesAlert.title,
+                                      message: Localization.DiscardChangesAlert.message,
+                                      preferredStyle: .alert)
+        alert.view.tintColor = .text
+
+        alert.addDestructiveActionWithTitle(Localization.DiscardChangesAlert.confirmActionTitle) { [weak self] _ in
+            self?.navigationController.dismiss(animated: true)
+        }
+
+        alert.addCancelActionWithTitle(Localization.DiscardChangesAlert.cancelActionTitle) { _ in }
+
+        if let popoverController = alert.popoverPresentationController {
+            popoverController.sourceView = navigationController.view
+            popoverController.sourceRect = navigationController.view.bounds
+            popoverController.permittedArrowDirections = []
+        }
```

Comment:
I don't think we need this, since the alert's preferred style is `.alert`. WDYT?

Reply:
Good point, removed in https://github.com/woocommerce/woocommerce-ios/pull/8009/commits/f6522590bf395c7f7838a6f31cc9c2423f03d8de